src-docs/src/views/card/card_footer.js

```diff
@@ -28,6 +32,7 @@ export default () => (
         icon={<EuiIcon size="xxl" type="devToolsApp" />}
         title="Developers Tools"
         description="Example of a short card description."
+        descriptionId="example.cards.developerTools.screenReader"
```

Comment:
We don't usually put decimals points in our strings for naming but do snake case instead like:
```suggestion
        descriptionId="exampleCardsDeveloperToolsScreenReader"
```